packages/std/src/exports.rs

```diff
@@ -21,6 +21,10 @@ use crate::{
     Env, HandleResponse, HandleResult, InitResponse, InitResult, QueryResponse, QueryResult,
 };
 
+#[cfg(feature = "staking")]
```

Comment:
Nice example.
After this is merged, we can remove the staking feature flag entirely in a new PR, right? Just include this export in the contract.
I mean, that's a large motivation of this as well.

Reply:
I don't know. Should contract developers be able to disable this or not?

Reply:
Poor wording on my part.

I would propose a follow-up PR to:
1. Move this `extern fn require_staking()` into `contracts/**/lib.rs` if they wish to use it (we can leave it commented out in the template)
2. Remove the `staking` feature flag from `cosmwasm-std`, we always have those types.

This makes the cosmwasm code simpler, but requires the contract developers to properly declare this if they want their contracts to have proper upload-time checks. We will need a similar approach for eg. terra custom messages. We cannot define those types in `cosmwasm-std`, as they are custom messages, but need to add those in the contracts.

Reply:
Maybe we can leave `staking`, but remove the flag from all data types (same layout always).

If there are helper functions to eg. do queries, these may be flagged, so people explicitly add an import. And then keep the `extern` here.

I would definitely like to remove feature flags from the CosmosMsg and QueryRequest types as this PR makes them unnecessary.

Reply:
Keeping the flag makes it easier for developers to toggle the export. Less copy/pasting of not so intuitive C function signatures.

Could you do the desired change as you are more familiar with this matter?

Reply:
okay, i will get into rust this afternoon and make some changes.